test/e2e/README.md

```diff
@@ -145,5 +146,9 @@ graph TD;
         end
     end
 ```
+
+##### Test Data for ZOT
```

Comment:
Just curious what is this for? Are the data under `testdata/zot/command/images` not "test data for ZOT"?

Reply:
Currently test data is partial, will update this section after I done migrating all the specs in command suite